pokecrystalvc.patch.template

```diff
@@ -0,0 +1,734 @@
+Type = 15
+
+[linkCable fake begin]
+Mode = 2
+Address = {{Address}}
+Type = 16
+
+[linkCable fake end]
+Mode = 2
+Address = {{Address}}
+Type = 17  
+
+;MURIYARI
+[linkCable block input]
+Mode = 2
+Address = {{Address}}
+Type = 18
+[linkCable block input2]
+Mode = 2
+Address = {{Address}}
+Type = 24
+[save game end]
+Mode = 2
+Address = {{Address}}
+Type = 20
+[term_exit]
+Mode = 2
+Address = {{Address}}
+Type = 25
+[room_check]
+Mode = 2
+Address = {{Address}}
+Type = 27
+[to_play2_mons1]
+Mode = 2
+Address = {{Address}}
+Type = 11
+[to_play2_trade]
+Mode = 2
+Address = {{Address}}
+Type = 12
+[to_play2_battle]
+Mode = 2
+Address = {{Address}}
+Type = 13
+[ret_heya]
+Mode = 2
+Address = {{Address}}
+Type = 14
+
+
+
+
+
+;ROM:3FBCD                 ld      b, $3E ; '>'
+;ROM:3FBCF                 inc     de
+;ROM:3FBD0                 call    unk_2D55
+;ROM:3FBD3                 xor     a
+;ROM:3FBD4                 ld      [byte_FFD2], a
+;ROM:3FBD6                 ld      [byte_FF4A], a
+;ROM:3FBD8                 call    unk_31C2
+;ROM:3FBDB                 call    unk_2FE2
+;
+;ROM:3FBCD:  06 3E 13 CD
+
+;0003fbb5h: 06 3E 13 CD                                     ; .>.?
+
+[fight begin]                        
+Mode = 11                       
+Type = 0                        
+Index = 1                       
+Address = {{ADDREss}}            
+Fixcode=a1:90   /* Not really sure what this fixcode does. */     
+              
+;12 1b 0b 79 b0  find next C9
+/*
+    This is ran whenever you enter the Hall of Fame.
+    It sets sMobileEventIndex and sMobileEventIndexBackup to $b.
+    essentially enabling the Celebi event.
+*/[BiographySave_ret]
+Mode = 2
+Address = {{Address}}
+Type = 60
+
+
+; print forbid 1
+;ROM:1758D                 ld      a, [byte_FFA9]
+;ROM:1758F                 and     2
+;ROM:17591                 jr      nz, unk_75B4
+;ROM:17593                 ld      a, [byte_FFA9]
+;ROM:17595                 and     1                      ;e6 01
+;ROM:17597                 jr      nz, unk_75A1
+;    
+; change "and 1" to "and 0"
+;00017595h: E6 01 20 08 CD BF 75 CD 2E 03 18 E9 FA 57 CE F5 ; ? .涂u?..辁W熙
+;00016c76h: E6 01 20 08 CD A0 6C CD 5A 04 18 E9 FA 63 CF F5 ; ? .蜖l蚙..辁c硝
+[print forbid 1]                                                   
+Mode = 1                                                                                                                  
+Address = {{address}}
+Fixcode={{patch::big}}      
+
+
+                            
+[print forbid 2]                                                                           
+Mode = 6                                                                                       
+Type = 0                                                                                       
+Address = {{address}}                                                                        
+MemAddress=0xffa7                                                                           
```

Comment:
This is already mentioned in mid-kid's first post on this PR; but I currently have `MemAddress` hardcoded to `0xffa7`. This is the address for `hJoyPressed::`. This comment is to serve as a reminder to fix make_patch.c to allow looking up non-prefixed symbols. 

This occurs in the following sections: [print forbid 2] [print forbid 3] [print forbid 5]

Reply:
So I am going to work make_patch one step at a time. I believe I have resolved the hJoyPressed hardcode problem. I modified make_patch.c to import all symbols.. Then I also added a command findaddress which will do a lookup for an address of a specific symbol that is not using a prefix or other patch naming convention. My C programming skills are super subpar, and there is a good chance I did some C programming no-no's... @mid-kid Could you please review https://github.com/pret/pokecrystal/pull/882/commits/099b2c7e378e5922893459f4776efc7a76834469 and let me know if I did anything bad.

Reply:
Also please check out https://github.com/pret/pokecrystal/pull/882/commits/fb7c391167c09c97cc208255c85fb94d068afd50 which modified "big" command to take in the labels from vc_patch and vc_patch_end.